docs/components/card.md

```diff
@@ -995,3 +995,46 @@ Card columns can also be extended and customized with some additional code. Show
   }
 }
 {% endhighlight %}
+
+## Publication
+Shows implementations of the publication element.
+
+{% example html %}
+<div class="card">
+  <article class="card-block card-block-action publication-block has-arrow">
+    <header class="publication-header">
+      <a class="publication-title" href="#" title="Personalsachbearbeiter/in in Grünwald" itemprop="title">Personalsachbearbeiter/in in Grünwald</a>
+    </header>
+    <section class="publication-meta">
+      <div class="meta-item meta-type">
+        <span class="badge badge-primary">Top Job</span>
+      </div>
+      <div class="meta-item meta-company" itemprop="hiringOrganization" itemscope="" itemtype="http://schema.org/LocalBusiness">
+        <span itemprop="name">FWU Institut für Film und Bild in Wissenschaft</span>
```

Comment:
Can we please use anonymised content for our styleguide, I know our mock API provides fake data for the search-publications endpoint: https://cnrkajfjqqx8frtfj-mock.stoplight-proxy.io/v4/search-publications maybe you can use this data.

In order to make reading the json response easier is recommend the JSON-Formatter Chrome Extension: https://github.com/callumlocke/json-formatter